portfolio/src/main/webapp/hobbies/script.js

```diff
@@ -3,15 +3,17 @@
 function addRandomSong() {
   
     const songs =
-      ['Good Days - SZA (R&B)', 'Pavane - Gabriel Faure (Classical)', 
-      'On Me - Lil Baby (Rap/Hip-Hop)', 'Eden - Brent Fiyaz (R&B)',
-      'Smerf Made This - G.T. (Rap/Hip-Hop)', 'Higher - Tems (R&B)'];
+      ['<iframe src="https://open.spotify.com/embed/track/3YJJjQPAbDT7mGpX3WtQ9A" width="300" height="80" frameborder="0" allowtransparency="true" allow="encrypted-media"></iframe>', 
+      '<iframe src="https://open.spotify.com/embed/track/4DLkSDc0HJ4i4jxjKq3Kuk" width="300" height="80" frameborder="0" allowtransparency="true" allow="encrypted-media"></iframe>',
+      '<iframe src="https://open.spotify.com/embed/track/4nCArjFjYyIioI6NGOSonr" width="300" height="80" frameborder="0" allowtransparency="true" allow="encrypted-media"></iframe>',
+      '<iframe src="https://open.spotify.com/embed/track/61gjdUMePfAOD5p0gnGvfv" width="300" height="80" frameborder="0" allowtransparency="true" allow="encrypted-media"></iframe>',
+      '<iframe src="https://open.spotify.com/embed/track/2QdSb68BzZGMgCbsrFmSLc" width="300" height="80" frameborder="0" allowtransparency="true" allow="encrypted-media"></iframe>'];
 
   // Pick a random greeting.
   const song = songs[Math.floor(Math.random() * songs.length)];
 
   // Add it to the page.
-  const greetingContainer = document.getElementById('greeting-container');
-  greetingContainer.innerText = song;
+  const SongContainer = document.getElementById('song-container');
+  SongContainer.innerHTML = song;
```

Comment:
`song-container` in HTML can be changed from a `div` to an `iframe` containing the other attributes (width, height, etc.), since those attributes are the same.
Change `songs` array into an array of src strings.
Pick a random song from `songs` array.
Set the song to the src attribute of `song-container` iframe element.
I know this is kinda long so feel free to ping me if you want further explanation.
Reference: https://www.w3schools.com/jsref/met_element_setattribute.asp